yggdrasil.go

```diff
@@ -134,14 +132,10 @@ func main() {
 	switch {
 	case *autoconf:
 		cfg = generateConfig(true)
-	case *useconffile != "" || *useconf:
```

Comment:
The reason `useconffile` was added is because it isn’t possible to redirect to stdin when running as a Windows service. By removing it, we would break the ability to run as a service on Windows using a static configuration file.

Reply:
Reverted this, but we should think about if there's a way to do this without useconffile. Maybe it can be wrapped in a script or something? I don't know much about how Windows services are configured/run.